src/transforms/globalCssToCssModule/globalCssToCssModule.ts

```diff
@@ -32,44 +40,87 @@ interface CodemodResult {
-        const filePath = sourceFile.getFilePath()
+export async function globalCssToCssModule(options: GlobalCssToCssModuleOptions): Promise<CodemodResult[]> {
+    const { project, shouldWriteFiles } = options
+    /**
+     * Find `.tsx` files with co-located `.scss` file.
+     * For example `RepoHeader.tsx` should have matching `RepoHeader.scss` in the same folder.
+     */
+    const itemsToProcess = project
+        .getSourceFiles()
+        .map(tsSourceFile => {
+            const tsFilePath = tsSourceFile.getFilePath()
+
+            const parsedTsFilePath = path.parse(tsFilePath)
+            const cssFilePath = path.resolve(parsedTsFilePath.dir, `${parsedTsFilePath.name}.scss`)
+
+            if (existsSync(cssFilePath)) {
+                return {
+                    tsSourceFile,
+                    cssFilePath,
+                }
+            }
+
+            return undefined
+        })
+        .filter(isDefined)
 
-        const parsedTsFilePath = path.parse(filePath)
-        const cssFilePath = path.resolve(parsedTsFilePath.dir, `${parsedTsFilePath.name}.scss`)
+    const codemodResultPromises = itemsToProcess.map(async ({ tsSourceFile, cssFilePath }) => {
+        const tsFilePath = tsSourceFile.getFilePath()
+        const parsedTsFilePath = path.parse(tsFilePath)
 
-        // TODO: add check if SCSS file doesn't exist and exit if it's not found.
         const sourceCss = readFileSync(cssFilePath, 'utf8')
-        const { css: cssModuleSource, filePath: cssModuleFileName } = await transformFileToCssModule(
+        const { css: cssModuleSource, filePath: cssModuleFileName } = await transformFileToCssModule({
             sourceCss,
-            cssFilePath
-        )
+            sourceFilePath: cssFilePath,
+        })
         const exportNameMap = await getCssModuleExportNameMap(cssModuleSource)
 
         processNodesWithClassName({
             exportNameMap,
-            nodesWithClassName: getNodesWithClassName(sourceFile),
+            nodesWithClassName: getNodesWithClassName(tsSourceFile),
         })
 
-        addClassNamesUtilImportIfNeeded(sourceFile)
-        sourceFile.addImportDeclaration({
+        addClassNamesUtilImportIfNeeded(tsSourceFile)
+        tsSourceFile.addImportDeclaration({
             defaultImport: STYLES_IDENTIFIER,
             moduleSpecifier: `./${path.parse(cssModuleFileName).base}`,
         })
 
-        // TODO: run prettier and eslint --fix over updated files.
-        return {
+        const result: CodemodResult = {
             css: {
                 source: cssModuleSource,
                 path: path.resolve(parsedTsFilePath.dir, cssModuleFileName),
             },
             ts: {
-                source: sourceFile.getFullText(),
-                path: sourceFile.getFilePath(),
+                source: tsSourceFile.getFullText(),
+                path: tsSourceFile.getFilePath(),
             },
         }
+
+        /**
```

Comment:
Cool descriptions. Only one thing that I've got a couple of times about /***/ jsdoc like comments. That they should be used only to describe some definitions or interfaces. For a simple logic description consider using // like comments. Also I think it could make sense to move/copy this description to `GlobalCssToCssModuleOptions`

Feel free to ignore this suggestion.

Reply:
@vovakulikov, thanks for the suggestion. Agree with copying the description to `GlobalCssToCssModuleOptions`. 
I've tested `//` comments in the editor, and it seems that they are not picked up by VS Code popups. 

<img width="328" alt="Screenshot 2021-08-31 at 17 26 05" src="https://user-images.githubusercontent.com/3846380/131521068-ccbe6ff5-923d-489f-881b-42732137cd3d.png">

vs

<img width="582" alt="Screenshot 2021-08-31 at 17 26 45" src="https://user-images.githubusercontent.com/3846380/131521075-38744917-92fe-47f8-b9c6-a7c34b558a6f.png">

Personal preference: I find `/***/` a bit easier to read for multiline comments. 
Let me if I miss other reasons to use `//` over `/***/`.